core/src/components/item/item.scss

```diff
@@ -281,6 +282,26 @@ button, a {
   box-sizing: border-box;
 }
 
+
+// Item Bottom
+// --------------------------------------------------
+
+.item-bottom {
+  @include margin(0);
+  @include padding(
+    0,
+    var(--inner-padding-end),
+    0,
+    calc(var(--padding-start) + var(--ion-safe-area-left, 0px) + var(--bottom-padding-start))
```

Comment:
`--bottom-padding-start` is not part of the public API, and defaults to `0px`, so do we even need to use it anymore?

Reply:
I'd feel comfortable removing it. Especially as another draft PR exposes `.item-bottom` as a shadow part that would allow developers to customize these styles in 6.1.

Is our stance that anything documented is public API and any other CSS variables are internal use and lower concern of affecting our release cycle?

Reply:
Yep exactly. If it's not in the comments above (this causes stencil to autogenerate it in the docs.json file) then it's internal.